web_ui/src/components/UsageBillingPage.tsx

```diff
@@ -110,8 +113,12 @@ function UsageAndBillingContainer({ children }: { children: React.ReactNode }) {
   )
 }
 
-function formatCents(cents: number): string {
-  return `\$${(cents / 100).toFixed(2)}`
+function formatCents(cents: number, currency: string): string {
```

Comment:
why aren't we using this function down below?

Reply:
fixed